src/components/residentPermit/EditResidentPermitPreview.tsx

```diff
@@ -59,7 +59,9 @@ const EditResidentPermitPreview = ({
           {t(`${T_PATH}.goBack`)}
         </Button>
         <Button
-          disabled={!(refundAccountNumber && isValidIBAN(refundAccountNumber))}
```

Comment:
Should support only Finnish IBAN

Reply:
I have updated the PR regarding this, but this line is not changed. So, what I did is creating our own version of `isValidIBAN` that checks if the IBAN number starts with "FI" and then calls the `isValidIBAN` function from` ibantools`.